asset-transfer-basic/rest-api-typescript/src/__tests__/api.test.ts

```diff
@@ -0,0 +1,738 @@
+/*
```

Comment:
Are there instructions somewhere about how to generate the test fixtures?

Reply:
I'm not sure what you mean- there aren't any generated test fixtures.